blocks/tabs/tabs.js

```diff
@@ -65,6 +78,17 @@ export default function decorate(block) {
     'aria-hidden': true,
   }, button(...title)));
 
+  if (isTabsCardsVariant) {
+    contents.forEach((content) => {
+      const tabsCardsBlock = buildBlock('tabs-cards', [[content.innerHTML]]);
```

Comment:
rather than instantiate a "tabs-cards" block, would it be possible to create a "cards" block instead for each "tabContents"?

Reply:
I don't think this makes sense since cards and tabs-cards are pretty different in terms of layout